config/config.php

```diff
@@ -7,7 +7,7 @@
      * By default, the `app` directory will be scanned recursively for models.
      */
     'directories' => [
-        base_path('app'),
+        base_path('app\Models'),
```

Comment:
Instead of `\`, it should have been `DIRECTORY_SEPARATOR` as it fails on Linux based machine. They use `/` instead of '\`.

Reply:
I created a follow-up PR: https://github.com/beyondcode/laravel-er-diagram-generator/pull/96